src/main/java/ca/uhn/fhir/jpa/starter/AuthenticationInterceptor.java

```diff
@@ -15,10 +19,10 @@
 
 @Interceptor
 public class AuthenticationInterceptor {
-	
+
     @Hook(Pointcut.SERVER_INCOMING_REQUEST_POST_PROCESSED)
-    public boolean incomingRequestPostProcessed(HttpServletRequest theRequest,
-            HttpServletResponse theResponse) throws AuthenticationException {
+    public boolean incomingRequestPostProcessed(HttpServletRequest theRequest, HttpServletResponse theResponse)
+            throws AuthenticationException, FileNotFoundException, IOException {
```

Comment:
since `FileNotFoundException` is a subclass of `IOException` , you can remove `FileNotFoundException` 
every ting else looks good